test.rb

```diff
@@ -0,0 +1,2 @@
+(1..100).detect {|i| i % 5 == 0 and i % 7 == 0 }
+(1..100).find {|i| i % 5 == 0 and i % 7 == 0 }
```

Comment:
Use `&&` instead of `and`.<br>Prefer `detect` over `find`.<br>Space between { and | missing.